CMakeLists.txt

```diff
@@ -28,7 +28,9 @@ if (PROJECT_IS_TOP_LEVEL)
   if(BUILD_NVFUSER_BENCHMARK)
     include(cmake/Dependencies.cmake)
   endif()
+  set(CMAKE_INSTALL_PREFIX "${NVFUSER_ROOT}/build")
```

Comment:
This doesn't quite feel like a place where we want to install things to. Build is just a build directory. If we are actually installing things during our standalone build, we might want to actually set things up properly later for our pip wheel installation. hint. we should build things into `./nvfuser`

Reply:
I tried that earlier but `nvfuser_tests` complained about finding `libc10.so`.